app/src/main/java/woowacourse/shopping/view/cart/CartPresenter.kt

```diff
@@ -50,7 +50,7 @@ class CartPresenter(
     }
 
     private fun convertIdToProductModel(cartProducts: List<CartProduct>) =
-        cartProducts.map { productRepository.find(it.id) }.map { it.toUiModel() }
+        cartProducts.asSequence().map { productRepository.find(it.id) }.map { it.toUiModel() }.toList()
```

Comment:
```suggestion
        cartProducts.map { productRepository.find(it.id).toUiModel() }
```
지금도 좋지만, 이렇게 작성해도 되지 않을까 싶네요. 😄